docs/bigcommerce-for-wordpress/setup/multi-site.md

```diff
@@ -30,7 +30,7 @@ Multiple sites can share the same API credentials or you can choose to create a
 

```

Comment:
Suggestion: line 22, add info theme to markdown for Stoplight Platform compatibility:
```
<!-- theme: info -->
```